docs/chain/collator.md

```diff
@@ -323,176 +297,175 @@ Secret phrase `very secure private key you should not use the example private ke
   SS58 Address:     4srC1aowD94H9UH9xsnfv7XV6oHU6dhCymKYZHWKsdddaP29
 ```
 
-The name of the file must be the public key prepended with `61757261` (hex representation of `aura`) and the content has to be the private key.
+The name of the file must be the *public* key prepended with `61757261` (HEX representation of `aura`) and without the `0x` prefix, while the content of the file has to be the secret phrase.
 
 ![](/img/chain/session-key-file.png)
 
-Path in the root chain folder `./keystores/61757261da3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d51010`
+For instance, with the keypair generated in the example, the session key file would be stored at the path `./keystores/61757261da3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d51010`.
 
 </TabItem>
 </Tabs>
 
-The collator has a session key by following the previous steps.
-The collator must then make an extrinsic by signing and submitting a transaction before the collator can begin collating on the network.
-Visit the [polkadot apps and connect to the peregrine network](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fperegrine.kilt.io#/explorer) (websocket address: `wss://peregrine.kilt.io`).
+Once a new session key is generated, the collator must then link that key to its own account in order to receive rewards for producing new blocks.
+This operation is performed by submitting a signed extrinsic to the blockchain.
+Visit the [PolkadotJS Apps and connect to the peregrine network](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fperegrine.kilt.io#/explorer) (websocket address: `wss://peregrine.kilt.io`).
```

Comment:
peregrine or spiritnet

Reply:
Yeah that's the thing. The whole initial part is about peregrine, and then only mentions spiritnet in its section. I would argue that it might be best to actually just put a small disclaimer in the beginning saying that the operations should be first tested out on peregrine, and perhaps provide the WS endpoint for that, but then only use spiritnet in the step-by-step guide. What do you think?

Reply:
would be cool to have a switch in the beginning of the doc and then we just show either peregrine or spiritnet data where appropriate. Like a Tab over the whole document. :D

Reply:
That would mean duplicating the same content twice, right? I would prefer to avoid that. But I will think about a better way to do that in a separate commit after addressing your comments and will ask you to review that.

Reply:
It would be cool to have switch on the top that switches between peregrine & spiritnet and that just replaces the values everywhere. 
I couldn't find something like that... But yes. Definitely don't put a tab for the whole doc! :D

Reply:
Let me know what you think https://github.com/KILTprotocol/docs/pull/25/commits/fc55280db6c574298a4fe65caf4e65087efe346e

Reply:
love it!